Sources/_StringProcessing/ConsumerInterface.swift

```diff
@@ -0,0 +1,602 @@
+      return consumeScalarProp(\.changesWhenCaseMapped)
+    case .changesWhenNFKCCasefolded:
+      return consumeScalarProp(\.changesWhenNFKCCaseFolded)
+    case .changesWhenLowercased:
+      return consumeScalarProp(\.changesWhenLowercased)
+    case .changesWhenTitlecased:
+      return consumeScalarProp(\.changesWhenTitlecased)
+    case .changesWhenUppercased:
+      return consumeScalarProp(\.changesWhenUppercased)
+    case .dash:
+      return consumeScalarProp(\.isDash)
+    case .deprecated:
+      return consumeScalarProp(\.isDeprecated)
+    case .defaultIgnorableCodePoint:
+      return consumeScalarProp(\.isDefaultIgnorableCodePoint)
+    case .diacratic: // spelling?
+      return consumeScalarProp(\.isDiacritic)
+    case .emojiModifierBase:
+      break // availability
+    case .emojiComponent:
+      break
+    case .emojiModifier:
+      break // availability
+    case .emoji:
+      break // availability
+    case .emojiPresentation:
+      break // availability
+    case .extender:
+      return consumeScalarProp(\.isExtender)
+    case .extendedPictographic:
+      break
+    case .fullCompositionExclusion:
+      return consumeScalarProp(\.isFullCompositionExclusion)
+    case .graphemeBase:
+      return consumeScalarProp(\.isGraphemeBase)
+    case .graphemeExtended:
+      break
+    case .graphemeLink:
+      break
+    case .hexDigit:
+      return consumeScalarProp(\.isHexDigit)
+    case .hyphen:
+      break
+    case .idContinue:
+      return consumeScalarProp(\.isIDContinue)
+    case .ideographic:
+      return consumeScalarProp(\.isIdeographic)
+    case .idStart:
+      return consumeScalarProp(\.isIDStart)
+    case .idsBinaryOperator:
+      return consumeScalarProp(\.isIDSBinaryOperator)
+    case .idsTrinaryOperator:
+      return consumeScalarProp(\.isIDSTrinaryOperator)
+    case .joinControl:
+      return consumeScalarProp(\.isJoinControl)
+    case .logicalOrderException:
+      return consumeScalarProp(\.isLogicalOrderException)
+    case .lowercase:
+      return consumeScalarProp(\.isLowercase)
+    case .math:
+      return consumeScalarProp(\.isMath)
+    case .noncharacterCodePoint:
+      return consumeScalarProp(\.isNoncharacterCodePoint)
+    case .otherAlphabetic:
+      break
+    case .otherDefaultIgnorableCodePoint:
+      break
+    case .otherGraphemeExtended:
+      break
+    case .otherIDContinue:
+      break
+    case .otherIDStart:
+      break
+    case .otherLowercase:
+      break
+    case .otherMath:
+      break
+    case .otherUppercase:
+      break
+    case .patternSyntax:
+      return consumeScalarProp(\.isPatternSyntax)
+    case .patternWhitespace:
+      return consumeScalarProp(\.isPatternWhitespace)
+    case .prependedConcatenationMark:
+      break
+    case .quotationMark:
+      return consumeScalarProp(\.isQuotationMark)
+    case .radical:
+      return consumeScalarProp(\.isRadical)
+    case .regionalIndicator:
+      break
```

Comment:
This property is a simple range `0x1F1E6...0x1F1FF` if we wanted to expose this.

Reply:
Let's expose any properties that lets you write grapheme breaking as a regular expression for testing purposes :-)